### DISCUSSION TITLE
Example in readme looks wrong?

### DISCUSSION THREAD

**Discussion by mrcjkb** [Frederick888/git-credential-keepassxc#100] [Q&A] [answered]
Hey :wave: 

I'm trying to set this up on NixOS (by manually editing the config).
The [configuration](https://github.com/Frederick888/git-credential-keepassxc#configuration) section in the readme
says to set

```
git config --global --replace-all credential.helper 'keepassxc --git-groups'
```

but there exists no `keepassxc --git-groups` option. This looks like an option of the `git-credential-keepassxc` CLI.
What is the correct command for setting this up?
Should it be `git config --global --replace-all credential.helper 'git-credential-keepassxc --git-groups get'`?

**Comment by Frederick888** [comment, marked as answer]
https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage#_under_the_hood

**Reply by mrcjkb** [reply]
Thanks :pray:
